Update Cloud Trail Permissions

harichintala1 opened the pull request.

davidkelliott commented on terraform/environments/bootstrap/member-bootstrap/iam.tf:
```diff
@@ -40,6 +40,10 @@ data "aws_iam_policy_document" "member-access" {
       "cloudtrail:CancelQuery",
       "cloudtrail:Create*",
+      "cloudtrail:StartLogging",
+      "cloudtrail:StopLogging",
```
```suggestion
```
Could we remove the StopLogging action please?  I'm wary of giving permission to turn off Cloudtrail logs! Thanks

harichintala1 replied:
removed 

      "cloudtrail:StopLogging",

harichintala1 commented:
@davidkelliott  I have removed stoptrail - however I have addded,

  "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?

davidkelliott commented:
> @davidkelliott I have removed stoptrail - however I have addded,
> 
> "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?

@harichintala1 I don't think we can have this either I'm afraid, we need to ensure that there is no way a Cloudtrail can be stopped or deleted for security reasons.  I think it will have to be if you need to remove a cloudtrail once created you will have to ask one of the MP engineers to manually do it.

harichintala1 commented:
> > @davidkelliott I have removed stoptrail - however I have addded,
> > "cloudtrail:DeleteTrail" - in view of when we remove resource from Terraform, it may fail to delete the Cloud trail we created ?
> 
> @harichintala1 I don't think we can have this either I'm afraid, we need to ensure that there is no way a Cloudtrail can be stopped or deleted for security reasons. I think it will have to be if you need to remove a cloudtrail once created you will have to ask one of the MP engineers to manually do it.

Hi - No problem, I have removed that now !

harichintala1 commented:
@davidkelliott could you review this for latest changes please !
